src/components/layouts/scrollLayout/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Suspense } from 'react';
+import { Outlet } from 'react-router-dom';
+
+import { Footer } from '@components/footer';
+import { Header } from '@components/header';
+import { PDScrollArea } from '@components/pdScrollArea';
+import { cn } from '@utils/helpers';
+
+interface ScrollLayout {
+  classNames?: string;
+  hasFooter: boolean;
+}
+
+export const ScrollLayout = (props: ScrollLayout) => {
+  const {
+    classNames,
+    hasFooter,
+  } = props;
+
+  return (
+    <Suspense>
+      <div className="grid max-h-screen min-h-screen grid-rows-layout overflow-hidden">
+        <Header />
+        <PDScrollArea className={cn(
```

Comment:
move classNames on new line ... this way if we add new props it will not make a lot of changes